pkg/virt-handler/isolation/detector.go

```diff
@@ -207,7 +209,28 @@ func setProcessMemoryLockRLimit(pid int, size int64) error {
 	return nil
 }
 
-func (s *socketBasedIsolationDetector) getPid(socket string) (int, error) {
+func GetPid(socket string) (int, error) {
```

Comment:
Can you give an example for what this does exactly? Also, can it be unit-tested?

Reply:
I didn't write it, just exported it. I don't even like the name, should be something like GetPeerPid(), but I can add documentation and maybe a unit test, although we will be testing the UDS kernel implementation.